addons/arcadian/data/arcadian_armour_plate.bisurf

```diff
@@ -0,0 +1,12 @@
+Density = 8600;
+Thickness = 16;
```

Comment:
Why are some capitalized and others not? Fairly sure they should all not be.

Reply:
Used the BI ones as reference to make these ones.

Reply:
Sure, but BI isn't known to set good standards. :P

Reply:
I'll lowercase them all.